libvips/foreign/tiffsave.c

```diff
@@ -605,10 +622,23 @@ vips_foreign_save_tiff_buffer_init( VipsForeignSaveTiffBuffer *buffer )
  * MINISBLACK TIFFs where black is a 0 bit, but if you set @miniswhite, it
  * will use 0 for a white bit. Many pre-press applications only work with
  * images which use this sense. @miniswhite only affects one-bit images, it
- * does nothing for greyscale images. 
+ * does nothing for greyscale images. Consider @bitdepth for depths 1, 2 and 4.
  *
  * Set @squash to squash 3-band float CIELAB images down to 8-bit CIELAB.
```

Comment:
This line needs to change too.

Reply:
Yes you are right but I didn't know how to properly change squash for the CIE stuff as commented above.